astropy/modeling/functional_models.py

```diff
@@ -1012,26 +1022,33 @@ class Voigt1D(Fittable1DModel):
 
     Parameters
     ----------
-    x_0 : float
+    x_0 : float or `~astropy.units.Quantity`
         Position of the peak
-    amplitude_L : float
-        The Lorentzian amplitude
-    fwhm_L : float
+    amplitude_L : float or `~astropy.units.Quantity`.
+        The Lorentzian amplitude (peak of the associated Lorentz function)
+        - for a normalized profile (integrating to 1), set
+        amplitude_L = 2 / (np.pi * fwhm_L)
+    fwhm_L : float or `~astropy.units.Quantity`
         The Lorentzian full width at half maximum
-    fwhm_G : float
+    fwhm_G : float or `~astropy.units.Quantity`.
         The Gaussian full width at half maximum
+    method : str, optional
+        Algorithm for computing the complex error function; one of
+        'Humlicek2' (default, fast and generally more accurate than ``rtol=3.e-5``) or
+        'Scipy', alternatively 'wofz' (requires `scipy`, almost as fast and reference in accuracy).
 
     See Also
     --------
     Gaussian1D, Lorentz1D
 
     Notes
     -----
-    Algorithm for the computation taken from
-    McLean, A. B., Mitchell, C. E. J. & Swanston, D. M. Implementation of an
-    efficient analytical approximation to the Voigt function for photoemission
-    lineshape analysis. Journal of Electron Spectroscopy and Related Phenomena
-    69, 125-132 (1994)
+    Either all or none of input ``x``, position ``x_0`` and the ``fwhm_*`` must be provided
+    consistently with compatible units or as unitless numbers.
+    Voigt function is calculated as real part of the complex error function computed from either
+    Humlicek's rational approximations (JQSRT 21:309, 1979; 27:437, 1982) following
+    Schreier 2018 (MNRAS 479, 3068; and `hum2zpf16m` from his cpfX.py module); or
```

Comment:
```suggestion
    Schreier 2018 (MNRAS 479, 3068; and ``hum2zpf16m`` from his cpfX.py module); or
```

Reply:
@dhomeier I think this is causing the error in RTD. With single quotes it's trying to link to an object `hum2zpf16m`.

Reply:
Good catch. I missed that warning, and it did not pop up in my local doc build. The other two warnings still persist, but since they are unrelated to any changes here, giving this a try!